### ISSUE TITLE
Liberty-Microprofile Stack - default port not clearly stated

### ISSUE THREAD

**Issue by lroseblade** [appsody/stacks#29]
Installing the Liberty-Microprofile stack results in the stack being installed and Liberty starting up, but does not clearly communicate that the default port is 9080 to those who are not familiar with Liberty (i.e. new users) . 

This should be more apparent at the end of the install (or on the appsody web site)

**Comment by neeraj-laad** [comment]
@lroseblade On our website, the tile for each stack has a `View on GitHub` link, These take you to the README for the stack. For `java-microprpfile` stack this would take you to https://github.com/appsody/stacks/tree/master/incubator/java-microprofile. 

I am trying to understand if you were:

- not able to find this link on our website (in which case we should look at the website design and make this more clearer).
or
- you found the README but it was not clear enough for you to provide this information. (in which case, we should look at updating the README)

**Comment by lroseblade** [comment]
Why require someone to make an additional click, go to another site, scroll through a readme to look for a port number? 

If you go here: https://appsody.dev/docs/getting-started/quick-start there's no natural user-flow that takes you anywhere where you can find the port number. It should be something a developer finds out passively, and not require the to search for it.

**Comment by neeraj-laad** [comment]
I agree making the user hunt for anything that is about getting them started is not good.

The intent was to use the default port for each language/framework, so the users find their apps in the place they expect them to be. (For example: MicroProfile this is 9080 and for Express this is 3000). Also, servers do usually print the list of URLs/port and other relevant information on the console when they start. 

If we trim down the amount of logs that user sees from OpenLiberty in this instance, this information will become more obvious for the user. Would that work?

**Comment by neeraj-laad** [comment]
Another possible option might be to let `stack providers` specify a welcome / getting started message in the stack, and this message can be shown to the user on their console when they run `appsody init <stack>` 

Some other tools like helm follow this model.

**Comment by lroseblade** [comment]
I don't mind which approach is taken providing it's obvious to the developer and doesn't assume they already know what the default port is for a runtime.

**Comment by neeraj-laad** [comment]
@Emily-Jiang

**Comment by Emily-Jiang** [comment]
@neeraj-laad when starting Liberty via liberty:run-server target, you will see the endpoint. appsody run should display the endpoint as well.

**Comment by nastacio** [comment]
Keep in mind that Appsody also maps the internal container ports to random port numbers via "-P" option, so that the stack output from inside the container is not a reliable way of determining the external port.
